samples/client/petstore/kotlin/retrofit2/src/main/kotlin/org/openapitools/client/auth/OAuthOkHttpClient.kt

```diff
@@ -0,0 +1,61 @@
+package org.openapitools.client.auth
```

Comment:
I think there are some files like this that don't belong to this PR.
Because the OAuth supports is still pending in the PR https://github.com/OpenAPITools/openapi-generator/pull/5697
Can you please generate the sample projects again?

Reply:
I forgott to mvn clean install before rerun generate all sample projects. Fixed with https://github.com/OpenAPITools/openapi-generator/pull/5718/commits/a9a01a4d01ba6366569ce82021773e8abb936da3